seacatauth/generic.py

```
@@ -117,12 +117,17 @@ def getseconds(self, key: str, default=None):
-	token_type, token_value = get_token_from_authorization_header(request)
-	if token_type == "Bearer":
-		return token_value
+	token = get_token_from_authorization_header(request)
+	if token is None:
+		L.debug("Request has no Authorization header")
+		return None
 
-	L.info("No Bearer token in Authorization header")
-	return None
+	token_type, token_value = token
+	if token_type != "Bearer":
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "token_type"

(S105)